app/src/main/res/drawable/darkened_automatic_circle_background.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+    <corners android:radius="200000dp"/>
```

Comment:
That's the idea, it automatically forms a circle. That works for everything smaller than the given radius. I can change it to the `selection_check_size` and it will work just fine for this scenario, but not generically anymore. Your call.

Reply:
it looks fine without the padding :)

Reply:
`selection_check_size` is okay